projects/ngrx.io/tools/examples/shared/boilerplate/cli/package-lock.json

```diff
@@ -9,32 +9,32 @@
       "version": "0.0.0",
```

Comment:
`package-lock.json` file could be delete now  ( check: https://github.com/ngrx/platform/pull/3436#pullrequestreview-987492791 )

Reply:
Good catch, thanks @cedricduffournet!